alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/transport/TransportGetAlertsAction.kt

```diff
@@ -78,6 +75,10 @@ class TransportGetAlertsAction @Inject constructor(
         getAlertsRequest: GetAlertsRequest,
         actionListener: ActionListener<GetAlertsResponse>
     ) {
+            val userStr = client.threadPool().threadContext.getTransient<String>(ConfigConstants.OPENDISTRO_SECURITY_USER_AND_ROLES)
```

Comment:
Can we always assume this setting is registered?

Reply:
If its not registered, it returns null. If you see otherwise please let me know.

Reply:
Makes sense, I'll test out when it is null and see how it works. Thanks!